public/bitemp_graph.js

```diff
@@ -260,20 +269,24 @@ var barChart = function() {
 
       split.append('text')
         .attr('class','tooltip-txt')
+        .style('opacity', 0)
+        .transition()
+        .duration(1500)
+        .style('opacity', 1)
         .style('text-anchor', 'middle')
         .attr('x', function(d) {
           var barx1 = xScale(moment(d.content.sysStart).toDate());
           var barx2;
           if (!d.content.sysEnd) {
-            return 0;
+            return 75;
           }
           if (d.content.sysEnd.indexOf('9999') === 0) {
             barx2 = xScale(moment(d.content.sysStart).add(5, 'y').toDate());
-            return (barx1+barx2)/2;
+            return (barx1+barx2)/2 + 60;
```

Comment:
This looks like hard-coding a fix for one data set. I think you're seeing that values with infinite sys ends are too far to the left, but I think this needs a better fix (or at least a comment with a TODO and an explanation for the 60).